lxd/ip/filter.go

```diff
@@ -0,0 +1,74 @@
+package ip
+
+import (
+	"github.com/lxc/lxd/shared"
+)
+
+// Action represents an action in filter
+type Action interface {
+	AddAction() []string
+}
+
+// ActionPolice represents an action of 'police' type
+type ActionPolice struct {
+	Rate  string
+	Burst string
+	Mtu   string
+	Drop  bool
+}
+
+// AddAction generates a part of command specific for 'police' action
+func (a *ActionPolice) AddAction() []string {
+	result := []string{"police"}
+	if a.Rate != "" {
+		result = append(result, "rate", a.Rate)
+	}
+	if a.Burst != "" {
```

Comment:
@presztak I think in this case we would normally have an empty line after the end of each if statement.